notebook/nbextensions.py

```diff
@@ -833,6 +844,35 @@ class DisableNBExtensionApp(ToggleNBExtensionApp):
     """
     _toggle_value = None
 
+    flags = _base_flags.copy()
+    flags['all'] = ({
+        "DisableNbExtensionApp" : {
+            "all_config_locations" : True,
+        }}, "Go through all config locations to disable loading an extension"
+    )
+
+    all_config_locations = Bool(False, config=True,
+        help="Go through all config locations to disable loading an extension",
+    )
+
+    def toggle_nbextension_python(self, module):
+        if self.all_config_locations:
+            for config_dir in jupyter_config_path():
+                disable_nbextension_python(self.section, module,
```

Comment:
We should make sure this does:

```python
if enabled at this level:
   set enabled = None # (not False. None pops config, False stores a value)
else:
   do nothing
```

So that it doesn't try to write any configuration files where there wasn't any to start with.